src/runtime_src/core/include/ert.h

```diff
@@ -626,28 +626,29 @@ struct cu_cmd_state_timestamps {
  * @ERT_START_NPU_PREEMPT: instruction buffer command with preemption format on NPU
  */
 enum ert_cmd_opcode {
-  ERT_START_CU          = 0,
-  ERT_START_KERNEL      = 0,
-  ERT_CONFIGURE         = 2,
-  ERT_EXIT              = 3,
-  ERT_ABORT             = 4,
-  ERT_EXEC_WRITE        = 5,
-  ERT_CU_STAT           = 6,
-  ERT_START_COPYBO      = 7,
-  ERT_SK_CONFIG         = 8,
-  ERT_SK_START          = 9,
-  ERT_SK_UNCONFIG       = 10,
-  ERT_INIT_CU           = 11,
-  ERT_START_FA          = 12,
-  ERT_CLK_CALIB         = 13,
-  ERT_MB_VALIDATE       = 14,
-  ERT_START_KEY_VAL     = 15,
-  ERT_ACCESS_TEST_C     = 16,
-  ERT_ACCESS_TEST       = 17,
-  ERT_START_DPU         = 18,
-  ERT_CMD_CHAIN         = 19,
-  ERT_START_NPU         = 20,
-  ERT_START_NPU_PREEMPT = 21,
+  ERT_START_CU             = 0,
+  ERT_START_KERNEL         = 0,
+  ERT_CONFIGURE            = 2,
+  ERT_EXIT                 = 3,
+  ERT_ABORT                = 4,
+  ERT_EXEC_WRITE           = 5,
+  ERT_CU_STAT              = 6,
+  ERT_START_COPYBO         = 7,
+  ERT_SK_CONFIG            = 8,
+  ERT_SK_START             = 9,
+  ERT_SK_UNCONFIG          = 10,
+  ERT_INIT_CU              = 11,
+  ERT_START_FA             = 12,
+  ERT_CLK_CALIB            = 13,
+  ERT_MB_VALIDATE          = 14,
+  ERT_START_KEY_VAL        = 15,
+  ERT_ACCESS_TEST_C        = 16,
+  ERT_ACCESS_TEST          = 17,
+  ERT_START_DPU            = 18,
+  ERT_CMD_CHAIN            = 19,
+  ERT_START_NPU            = 20,
+  ERT_START_NPU_PREEMPT    = 21,
+  ERT_START_NPU_PDI_IN_ELF = 22,
```

Comment:
Let's start with a different name: ERT_START_NPU_PREEMPT_ELF. (With this flow, PDI is not necessarily in ELF.)

Reply:
sure Larry